src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -20,39 +21,51 @@ import { NoticeProps, MenuListComponentProps, MenuProps } from 'react-select/lib
 import { MultiValueProps } from 'react-select/lib/components/MultiValue';
 import { getOptionValue, getOptionLabel } from 'react-select/lib/builtins';
 import { ControlProps } from 'react-select/lib/components/Control';
+import { IndicatorProps } from 'react-select/lib/components/indicators';
 import { StylesConfig } from 'react-select/lib/styles';
 import { SelectComponents } from 'react-select/lib/components';
 
 import './Autocomplete.scss';
 
-const styles = ({ spacing, palette }: Theme) =>
+const styles = ({ spacing, palette, transitions }: Theme) =>
   createStyles({
     input: {
+      height: 'auto',
       display: 'flex',
       '& > div:first-child > div:last-child': {
         marginTop: 0,
         marginBottom: 0,
         paddingTop: 0,
-        paddingBottom: 0
+        paddingBottom: 0,
+        lineHeight: 'inherit'
+      },
+      '& > div:last-child': {
+        cursor: 'pointer'
       },
       '& > div:last-child > span': {
         backgroundColor: 'transparent'
+      },
+      '& > div:last-child > div': {
+        paddingTop: 0,
+        paddingBottom: 0,
+        height: 19
       }
     },
     valueContainer: {
       display: 'flex',
       flexWrap: 'wrap',
       alignSelf: 'center',
+      lineHeight: 'inherit',
       flex: 1,
-      alignItems: 'center'
+      alignItems: 'center',
+      position: 'relative'
```

Comment:
This fixes the issue with the ellipsis. The select container was taking the width of the autocomplete container and not the width of the valueContainer. By adding relative positioning, it then causes the child value container to take its parents (valueContainer) width.